packages/ses/CHANGELOG.md

```diff
@@ -3,6 +3,27 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.5.0](https://github.com/endojs/endo/compare/ses@1.4.1...ses@1.5.0) (2024-05-06)
+
+
+### Features
+
+* **ses:** add COMPARTMENT_LOAD_ERRORS env option to print verbose output for module loading errors ([eca5f23](https://github.com/endojs/endo/commit/eca5f235358fcf1fe135afca56a2f0a6056bb1b2))
```

Comment:
Looking at this, I see a function named `throwAggregateError` which throws a `TypeError`, not and `AggregateError`. Perhaps this predates our support for `AggregateError`? Attn @naugtur

Reply:
https://github.com/endojs/endo/blob/b09f8c05620b28df17487a0de9fc2098d7f20eee/packages/ses/src/module-load.js#L366

This does predate SES exposing AggregateError. We do not yet _rely_ on AggregateError, though it may be time to consider doing so.

Reply:
@naugtur @kriskowal , could you file an issue?

@kriskowal no need to delay this release on this issue. Resolving.